pkg/record/diskrecorder/diskrecorder.go

```diff
@@ -245,6 +245,9 @@ func (r *Recorder) PeriodicallyPrune(ctx context.Context, reported AlreadyReport
 				}
 				return true, nil
 			})
+			if err != nil {
+				klog.V(4).Infof("Fail to properly prune last report within %s: %v", interval.Truncate(time.Minute), err)
```

Comment:
Is it a minute? wait.Backoff{Duration: time.Second, Steps: 4, Factor: 1.5} I am not sure how the backoff works but I guess the worst case (when the timeout exceeds) is 1s * 1,5 * 1.5 * 1.5 * 1.5 in this case or maybe just 1.5 * 4 ...not sure.

Reply:
As I told you on the channel, I tested it, and Minutes works fine.